_integration-schemas/codat/v1/bills.md

```diff
@@ -9,7 +9,9 @@ singer-schema: "https://github.com/singer-io/tap-codat/blob/master/tap_codat/sch
 description: |
   The `{{ table.name }}` table contains info about the bills in your {{ integration.display_name }} account. A bill is an itemized record of goods or services purchased from a [supplier](#suppliers).
 
-replication-method: "Full Table"
+replication-method: "Key-based Incremental"
+
+replication-key: "modifiedDate"
```

Comment:
This isn't the correct YAML format for doing rep keys this way - you're missing something.